src/app/inputs/inputs.component.ts

```diff
@@ -1,44 +1,42 @@
-import { Component } from '@angular/core';
+import { Component, EventEmitter, Output } from '@angular/core';
 import { ApiService } from '../services/api.service';
-import { Task } from '../interface/task';
+import { Task } from '../interfaces/task.interface';
 
 @Component({
   selector: 'app-inputs',
   templateUrl: './inputs.component.html',
   styleUrls: ['./inputs.component.css']
 })
 export class InputsComponent {
-  
 
   constructor(private apiService: ApiService) { }
   
   addTask() {
-
     const inputTitle = document.getElementById('input-ul') as HTMLInputElement | null;
     const taskTitle = inputTitle?.value;
     const inputDesc = document.getElementById('desc-ul') as HTMLInputElement | null;
     const taskDesc = inputDesc?.value;
+
     if (!taskTitle || !taskDesc) {
```

Comment:
Using angular forms would be a better way to check if some of the inputs fields are empty.
It has an element called validator that would be relevant to use in this case.